### PR TITLE
#2086 - Enhance regulatory body dropdown

### PR DESCRIPTION
**Pull request by dheepak-aot**
## New dropdown item Senate, Academic Council, Education Council, and/or Program Council and Board of Governors

- [x] Add a new dropdown item called "Senate, Academic Council, Education Council, and/or Program Council and Board of Governors". Add the dropdown item to institution profile and program both.
- [x] Define the option code as senateOrEducationCouncil.
- [x] In case of other regulatory body is selected, the text input to enter other regulatory body has been moved under the regulatory body dropdown.

### Create/Edit Program

image

Other
image

### Institution Profile (Edit from ministry)
image

image

image

###  Institution Setup (Ignore the label)
image

image

### CODE REVIEW

**Review by sh16011993 — APPROVED** [review]
LGTM @dheepak-aot  👍

File: sources/packages/forms/src/form-definitions/institutionprofile.json [hunk 1]
```diff
@@ -562,216 +562,186 @@
...
+          "id": "e5eqny",
+          "defaultValue": ""
+        },
+        {
+          "autofocus": false,
+          "input": true,
+          "tableView": true,
+          "inputType": "text",
+          "inputMask": "",
+          "label": "Other institution regulating body",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/institutionprofile.json:713]: The last PR raised the question (answered by @hellolynn-tbtb) about using `Regulating` vs `Regulatory` and apparently the expected way would be `Regulatory` and per the wiki below.
The institution profile is right but apparently the programs are not.
@JasonCTang does it make sense? Should we adjust the programs?

image
Source: https://github.com/bcgov/SIMS/wiki/Acronyms-&-Terminology#word--spelling-usage

File: sources/packages/forms/src/form-definitions/institutionprofilecreation.json [hunk 2]
```diff
@@ -1025,217 +1025,188 @@
...
-              ],
-              "width": 6,
-              "offset": 0,
-              "push": 0,
-              "pull": 0
-            }
-          ],
-          "type": "columns",
-          "hideLabel": true,
+          "label": "Institution regulating body",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/institutionprofilecreation.json:1028]: Sane from https://github.com/bcgov/SIMS/pull/2088/files#r1254984989

File: sources/packages/forms/src/form-definitions/institutionprofilecreation.json [hunk 3]
```diff
@@ -1025,217 +1025,188 @@
...
+          "defaultValue": "",
+          "isNew": false
+        },
+        {
+          "autofocus": false,
+          "input": true,
+          "tableView": true,
+          "inputType": "text",
+          "inputMask": "",
+          "label": "Other regulating body",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/institutionprofilecreation.json:1177]: Same from https://github.com/bcgov/SIMS/pull/2088/files#r1254984989

File: sources/packages/forms/src/form-definitions/institutionprofile.json [hunk 4]
```diff
@@ -562,216 +562,186 @@
           "defaultValue": null
         },
         {
+          "label": "Institution regulating body",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/institutionprofile.json:565]: Same from https://github.com/bcgov/SIMS/pull/2088/files#r1254984989

**Review by ann-aot — COMMENTED** [review]
pls take a look at the devs' comments

**Review by guru-aot — APPROVED** [review]
LGTM, nice work

**Comment by dheepak-aot** [comment]
> pls take a look at the devs' comments

Just letting know @ann-aot that comment is not part of the PR. But I strongly agree with the idea of making this label change now rather than doing a hotfix/update on production later.

**Comment by ann-aot** [comment]
> > pls take a look at the devs' comments
> 
> Just letting know @ann-aot that comment is not part of the PR. But I strongly agree with the idea of making this label change now rather than doing a hotfix/update on production later.

I am sorry, I didn't get it. So, the plan is not to do the changes in this PR?

**Comment by dheepak-aot** [comment]
> > > pls take a look at the devs' comments
> > 
> > 
> > Just letting know @ann-aot that comment is not part of the PR. But I strongly agree with the idea of making this label change now rather than doing a hotfix/update on production later.
> 
> I am sorry, I didn't get it. So, the plan is not to do the changes in this PR? @dheepak-aot

It is the opposite. To do the change as discussed.

File: sources/packages/web/src/views/aest/institution/Profile.vue [hunk 5]
```diff
@@ -56,12 +56,12 @@
               :propertyValue="institutionProfileDetail.institutionTypeName"
             />
             <title-value
-              propertyTitle="Regulating body"
+              propertyTitle="Regulatory body"
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/aest/institution/Profile.vue:59]: Thanks for changing these ones also.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the additional changes, looks good 👍

**Review by ann-aot — APPROVED** [review]
Thanks for including the label change in the PR. Good work @dheepak-aot  👍

**Pull request merged**